packages/rest-specs/specs/resiliency/srv-driven/main.tsp

```diff
@@ -0,0 +1,169 @@
+import "@typespec/http";
```

Comment:
lets omit resiliency as well, and any spec that use anything from `typespec-client-generator-core`

Reply:
Done